### PR TITLE
Enforce MPD setup during installation

### PR DESCRIPTION
**Pull request by AlvinSchiller**
Following the report of #2306:

- [x] The user option for MPD setup is removed. Instead the mpd.conf is always deployed, as Phoniebo is not functionally otherwise. The manual process to fix this is not straight forward and error prone.
- [x] Multiple sample conf files existed in the repo. This was cleaned to only have one left used by the setup.

TODO:
- [x] manual installation tests on device

### CODE REVIEW

File: scripts/installscripts/install-jukebox.sh [hunk 1]
```diff
@@ -1097,33 +1067,31 @@ install_main() {
...
-        # start mpd
-        echo "Starting mpd service..."
-        sudo service mpd restart
-        sudo systemctl enable mpd
-    fi
+    echo "Configuring MPD..."
+    local mpd_conf="/etc/mpd.conf"
+    # MPD configuration
+    # -rw-r----- 1 mpd audio 14043 Jul 17 20:16 /etc/mpd.conf
+    sudo cp "${jukebox_dir}"/misc/sampleconfigs/mpd.conf.buster-default.sample ${mpd_conf}
```
Comment (s-martin) [inline scripts/installscripts/install-jukebox.sh:1075]: We could rename this file removing the outdated buster reference

**Comment by s-martin** [comment]
> @s-martin 
> Additional change suggestion (already checked in but can be reverted):
> - Currently the spotify installation defaults to "yes". As spotify is not working for some time now, without user changes, the default should be changed to "no", having the "classic" edition as default.

Yes!

File: misc/sampleconfigs/mpd.conf.sample [hunk 2]
```diff
@@ -227,14 +255,14 @@ audio_output {
 #	device		"/dev/dsp"	# optional
 #	mixer_type      "hardware"      # optional
 #	mixer_device	"/dev/mixer"	# optional
-#	mixer_control	"%AUDIOiFace%"		# optional
+#	mixer_control	"PCM"		# optional
```
Comment (s-martin) [inline misc/sampleconfigs/mpd.conf.sample:258]: is this intended?

**Review by s-martin — APPROVED** [review]

File: misc/sampleconfigs/mpd.conf.sample (hunk 2 above)
Comment (AlvinSchiller) [reply misc/sampleconfigs/mpd.conf.sample:258]: Thats a display problem due to the renaming and not a real change.
The obsolete and outdated mpd.conf file that was removed before, had also the name "mpd.conf.sample".
In the overall change view this is now display as changes, rather deletion and renaming.

If you just view the last commit you can see no changes of content and just the renaming of the file.

**Pull request merged**